packages/amazonq/package.json

```diff
@@ -567,6 +588,36 @@
                 "command": "aws.amazonq.showHistoryInHub",
                 "title": "Show Job Status",
                 "icon": "$(history)"
+            },
+            {
+                "command": "aws.amazonq.selectCustomization.commandPalette",
+                "title": "%AWS.codewhisperer.customization.notification.new_customizations.select%",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "aws.codewhisperer.connected && aws.codewhisperer.connectionType != builderId"
+            },
+            {
+                "command": "aws.amazonq.gettingStarted",
+                "title": "Try inline suggestion examples",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "aws.codewhisperer.connected"
+            },
+            {
+                "command": "aws.amazonq.openChatPanel.commandPalette",
+                "title": "Open Chat Panel",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "aws.codewhisperer.connected"
+            },
+            {
+                "command": "aws.amazonq.toggleCodeSuggestion.commandPalette",
+                "title": "%AWS.amazonq.toggleCodeSuggestion%",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "aws.codewhisperer.connected"
+            },
+            {
+                "command": "aws.codeWhisperer.toggleCodeScan.commandPalette",
+                "title": "%AWS.amazonq.toggleCodeScan%",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "aws.codewhisperer.connected && aws.codewhisperer.connectionType != builderId"
```

Comment:
there should be no need for a new `aws.codewhisperer.connectionType` global. a "toggle" command always looks the same in the palette, it's just named `Toggle Foo`. It shouldn't change its wording based on state, that is confusing, and also requires these extra global flags.